<!--?php include 'header.php'; ?--><!doctype html><html><head>
    <meta http-equiv="Content-Type" content="text/html, charset=UTF-8">
    <title>Ecological Archives E095-124-metadata</title>
    
    <script type="text/javascript" src="http://www.esapubs.org/templates/scripts/analytics.js"></script>
    <style type="text/css">.sectionHeader {font-size: 1.1em; font-variant: small-caps; font-weight: bold;}</style>
    <style type="text/css">
body {background-color: #FFF; font-family: Times New Roman, Times, serif;}

hr {
	height: .15em;
	color: gray;
	background-color: gray;
	border-width: 0;
}
hr.divider {
	width: 50em;
	margin-top: 1.7em;
	margin-bottom: 2em;
	margin-left: 0;
}

p {max-width: 50em;}

li {max-width: 48em;}

sub, sup {font-size: .7em;}

table, td {
	border: 1px solid black;
	border-collapse: collapse;
	text-align: center;
    vertical-align: top;
}
td {max-width: 70em; padding: .1em;}
table, tr {
	padding: .1em;}
tr:first-child {font-weight: bold;}

table.equation {border: none; width: 48em;}
.equation td {border: none; text-align: left; vertical-align: middle;}
.equation tr:first-child {font-weight: normal;}
.equation td:last-child {text-align: right; width: 3em;}

table.noFormat {border: none; text-align: inherit;}
.noFormat td {border: none; text-align: inherit;}
.noFormat tr:first-child {font-weight: normal;}

/* Position modifiers.  Use these to alter placement of elements on the page. */

.center {margin-left: auto; margin-right: auto; text-align: center;}
.textCenter {text-align: center;}
.textLeft  {text-align:left;}

.indent {margin-left: 2em;}
p.indent {max-width: 48em;}
p.heading.indent {max-width: 40em;}
.halfIndent {margin-left: 1em;}

/* Font modifiers.  Use these to alter font style and type. */

.courier {font-family: Courier new, Courier, mono;}
.heading {max-width: 45em; margin-top: .7em; font-size: 1.2em; font-weight: bold;}
.overline {text-decoration: overline;}
.over {text-decoration: overline;}
.roman{font-style: normal; text-decoration: none; font-weight: normal;}
.smallcaps {font-variant: small-caps;}

#links {font-size: .7em; font-family: Arial, Helvetica, sans-serif;}
</style></head>
    <body bgcolor="#FFFFFF">
        <p class="heading center"><i>Ecological Archives</i> E095-124-D1</p>
        <p class="heading">Benjamin Planque, Raul Primicerio, Kathrine Michalsen, Michaela Aschan, Grégoire Certain, Padmini Dalpadado, Harald Gjøsæater, Cecilie Hansen, Edda Johannesen, Lis Lindal Jørgensen, Ina Kolsum, Susanne Kortsch, Lise-Marie Leclerc, Lena Omli, Mette Skern-Mauritzen, Magnus Wiedmann. 2014. Who eats whom in the Barents Sea: a food web topology from plankton to whales. <i>Ecology</i> 95:1430. <a href="http://dx.doi.org/10.1890/13-1062.1">http://dx.doi.org/10.1890/13-1062.1</a></p>
        <!-- Content starts here!  Don't delete anything before this! -->
        <hr>
        <p class="sectionHeader"><a href="#update">UPDATED METADATA (ADDED AUGUST 2015)</a></p>
        <p class="sectionHeader">Metadata</p>

        <p class="sectionHeader">Class I. Data set descriptors</p>
        <p><b>Abstract:</b> A food web is an ecological network and its topological description consists of the list of nodes, i.e., trophospecies, the list of links, i.e., trophic interactions, and the direction of interactions (who is the prey and who is the predator). Food web topologies are widely used in ecology to describe structural properties of communities or ecosystems. The selection of trophospecies and trophic interactions can be realized in different manners so that many different food webs may be constructed for the same community. In the Barents Sea, many simple food webs have been constructed. We present a comprehensive food web topology for the Barents Sea ecosystem, from plankton to marine mammals. The protocol used to compile the data set includes rules for the selection of taxa and for the selection and documentation of the trophic links. The resulting topology, which includes 244 taxa and 1589 trophic links, can serve as a basis for topological analyses, comparison with other marine ecosystems, or as a basis to build simulation models of the Barents Sea ecosystem. The data set consists of three related tables: (1) the list of taxa, (2) the list of pairwise interactions, and (3) the list of bibliographical references.        </p>
        <p>
          <b><i>Key words</i></b>: <i>benthos; birds; fish; mammals; plankton; trophic interactions.</i>
        </p>
        <p><b>Data files</b></p>
        <blockquote>
          <p class="courier">SpeciesList.txt</p>
          <p class="courier">PairwiseList.txt</p>
          <p class="courier">References.txt</p>
        </blockquote>
        <p>&nbsp;</p>
        <p class="sectionHeader">Class II. Research origin descriptors</p>
        <p><b>A. Overall project description:</b></p>
        <p><i>A.1. Identity:</i></p>
        <p>This data set was compiled as part of the project entitled BarEcoRe: Barents Sea Ecosystem Resilience under global environmental change</p>
        <p><i>A.2. Originators:</i></p>
        <p>The project was coordinated by Benjamin Planque at the Institute of Marine Research, Norway, benjamin.planque@imr.no. The construction of the topology was initiated during a workshop, held in Bergen and Tromsø on the 8–9 June 2011.</p>
        <p><i>A.3. Period of study:</i></p>
        <p>BarEcoRe started in June 2010 and ended in May 2013. The food web topology data set was assembled during that period. The topology does not refer to a period in particular but is based on documentation and observations throughout the 20th and early 21st century.</p>
        <p><i>A.4. Objectives:</i></p>
        <p>The main objective of BarEcoRe was to evaluate the effects of global environmental change on the future structure and resilience of the Barents Sea ecosystem. The food web topology was constructed in order to investigate the structural properties of the Barents Sea food web and how these may be related to ecosystem resilience.</p>
        <p><i>A.5. Abstract:</i></p>
        <p>BarEcoRe aimed to evaluate the effects of global environmental change on the future structure and resilience of the Barents Sea ecosystem. This was studied by investigating the effects of past changes in climate and fisheries on the Barents Sea ecosystem, by developing indicators of ecosystem resilience, diversity and structure, and by forecasting the possible future states of the Barents Sea ecosystem under particular environmental and fisheries scenarios. The key questions addressed by the project were: (1) What are the key characteristics of past temporal and spatial variations in fish and benthos communities and how are these related to past climate variability and fishing pressure? (2) How does climate variability and change propagate through the Barents Sea ecosystem and influence species  interactions? (3) How can the combined effects of fisheries and climate modify the spatial distribution of plankton, benthos, and fish species in the Barents Sea? (4) What determines vulnerability or resilience of the Barents Sea ecosystem and how will these be affected by possible future changes in climate and fisheries regimes? (5) Can we detect early warning signals and can we evaluate management strategies with regards to ecosystem resilience? These questions were studied through a number of approaches which included: process studies of trophic interactions, retrospective analysis of plankton, benthos, and fish communities' structure and ocean climate, predictive models of ocean climate, population spatial distribution models under climate and biological constraints and ecological network analysis. This data set is the basis for the network analysis. </p>
        <p><i>A.6. Sources of funding:</i></p>
        <p>BarEcoRe was funded by the Norwegian Research Council (NFR-grant 200796/41), the Institute of Marine Research, Norway, and the University of Tromsø, Norway. Additional funding for the construction of this data set was received from the Tromsø Research Foundation. </p>
        <p><b>B. Research motivation</b></p>
        <p>The study of food webs has been a topic of ecological research for many decades. The growing interest in the study of complex systems and the use of graph theory to investigate properties of large networks has vitalised food web research and the analysis of food web structures or topologies (Dunne 2009, Petchey et al. 2010). A food web consists of throphospecies (one or more species sharing predators and prey) and the feeding links between them (Yodzis 1988). Early work on the comparison of food webs between areas and ecological systems was fostered by the ECOWeB data bank, a compilation of food webs (Cohen 1989). However, such compilations suffered from lack of resolution and great disparities in the methods used to assemble  food webs from different systems. This led to specific recommendations for  improving food web construction (Cohen et al. 1993). Re-analyses of the original ECOWeB data revealed that incomplete food web topologies led to inaccurate  conclusions (Goldwasser and Roughgarden 1997). To be useful for comparative analyses, food webs must be constructed using a clearly described methodology (Jordán 2003). Currently, many food webs exist for the Barents Sea (see, e.g., , Sakshaug et al. 2009). However these are usually restricted to few taxa (typically less than 20), with a particular focus on few groups (e.g., , fish and mammals) and their method of construction is poorly documented. The largest food web topology for the Barents Sea was recently assembled by Bodini et al. (2009) for the purpose of investigating secondary extinctions. It contains 254 species, of which 151 have one or more trophic links. The method and data sources used to construct this topology are not documented and it is therefore uncertain if such a data set can be used to derive reliable ecological conclusions on the functioning of the Barents Sea ecosystem, although this has already been attempted (de Santana et al. 2013). The motivation for compiling the current data set is to provide the first comprehensive food web topology for the Barents Sea for which the methodology is transparent and the data source explicit. We are aware of the large body of relevant references in Russian that have only partly been included in the present work. Future revisions of this topology will likely expand on the number of nodes, links and bibliographic references. </p>
        <p><b>C. General methodology:</b></p>
        <p><i>C.1. System definition</i></p>
        <p>The food web topology is defined for the ecosystem of the Barents Sea from plankton to sea mammals. The topology consists of 3 elements: nodes, i.e., trophospecies, links, i.e., trophic interactions, and directions, i.e. who is the predator and who is the prey.</p>
        <p><i>C.2. Spatial and temporal coverage</i></p>
        <p>The spatial coverage of our data set extends over the Barents Sea, limited by Novaya Zemlya in the East. The Barents Sea is a shelf sea situated north of Norway and Russia extending from 68.5°N to 82.58°N and from 8.0°E to 68.5°E. It covers an area of 1.6 million km² with an average depth of 230 m (Carmack et al. 2006). The temporal coverage of the data set is set by the literature review and available data and covers the period 1927–2012, although half of the references used are from the last two decades (1994–2012).</p>
        <p><i>C.3. Data sources</i></p>
        <p>There are three primary sources for the data set: (1) peer reviewed publications; (2) gray literature and institutional reports; (3) inference on the basis of knowledge on similar species or comparable regions. Norway and Russia have, through the last 25 years, collected an extensive amount of fish-stomach data from the Barents Sea that was used to document links between fish and their prey. No surveys were conducted within the BarEcoRe project. The references used to support the inclusion of the trophic links, are provided in the data set (see section C.5.d).</p>
        <p><i>C.4.  Nodes, trophospecies</i></p>
        <p>Nodes in the food web topology correspond to trophospecies. A trophospecies is one or more species sharing predators and prey. When possible, individual trophospecies correspond to taxonomic species. Yet, these can sometimes refer to larger taxonomic groups (e.g., , genus, family, class) or to specific stages (larvae, juveniles, adults). By default, a trophospecies is given with reference to the adult stage.</p>
        <p><i>C.4.a: selection criteria</i></p>
        <p>Trophospecies selection was done using four criteria: (1) there is information on trophic interactions, i.e., prey and/or predator/s are known; (2) the spatial extent of the species exceeds 5% of the total Barents Sea area; (3) the temporal extent is sufficient, i.e., the trophospecies has been observed in several instances over several years or seasons; (4) if the trophospecies has not been observed in the Barents Sea, but its distribution can expand into the area due to climate change (e.g., , mackerel <i>Scomber scombrus</i>) it can also be included. </p>
        <p><i>C.4.b: grouping and splitting</i></p>
        <p>When taxonomic identification was not conducted to species level, trophospecies may refer to larger groups such as genus, family, or class. There are few instances where different taxon levels result in overlap in the data set: <i>Oikopleura</i> sp., <i>Pagurus</i> sp., <i>Astarte</i> sp., <i>Sebastes</i>_sp, <i>Ammodytes</i>, Actinaria, and Porifera. Detritus forms a group which has no taxonomic identity. Many marine organisms undergo ontogenetic metamorphosis and/or changes in size that may span over several orders of magnitude. In such cases, the trophic links (relevant prey and predators) can vary greatly during the lifetime of the organisms. A group of fish eggs and larvae is therefore defined.</p>
        <p><i>C.4.c: seasonal and spatial variations</i></p>
        <p>The spatial distribution of many trophospecies is limited to specific regions of the Barents Sea and these can vary with seasons and years. As a result, all trophospecies listed in this topology do not necessarily meet. The resulting topology is a reference for all Barents Sea species, but analyses of the food web structure must take into account when and where prey and predators actually interact. </p>
        <p><i>C.4.d: phylogenetic information</i></p>
        <p>For each trophospecies listed in the data set, we provide additional information on order, class and phylum or sub-phylum. The information is derived from the World Register of Marine Species (WoRMS: <a href="http://www.marinespecies.org/">http://www.marinespecies.org/</a>) and Fish Base (http://www.fishbase.us/).</p>
        <p><i>C.4.e: naming of trophospecies</i></p>
        <p>Naming of trophospecies was done using the latin names for genus and species, when applicable. For higher systematic taxa, the latin name for genus, family, order, class, or phylum was used. For specific stages or groups that did not correspond to a taxon, a self-explanatory english name for the trophospecies was chosen (e.g., , detritus).</p>
        <p><i>C.5. Links and directions, trophic interactions</i></p>
        <p>Food web links are trophic interactions between two trophospecies. The relationship is directional from the prey to the predator. In case of cannibalism, the prey and predator is the same trophospecies. Basal species are trophospecies without prey. Top predators are trophospecies without predators. In the present data set, each trophospecies has at least one link towards a prey or a predator, due to the selection criteria (<i>C.4.a, </i>1).</p>
        <p><i>C.5.a: selection criteria</i></p>
        <p>In the present data set only the effective trophic links were reported by specifying prey and predator names, i.e., there is no information to support the absence of trophic links. A trophic link between two trophospecies was included in the data set when there was information available to support the link. The information could come in various forms, from published material in peer reviewed journals, survey reports, institutional databases, expert knowledge or inference on the basis of knowledge on similar species or comparable regions. The information supporting individual links was qualified using a 1–4 scale, as detailed in section C.5.c.</p>
        <p><i>C.5.b: seasonal and spatial variations</i></p>
        <p>When a link is documented, it indicates that a specific prey is known to be eaten by a specific predator. As the spatial distribution of prey and predators can vary temporally within the Barents Sea, it does not imply that prey and predator are interacting everywhere and at all times. As a result, all trophic links listed in this topology can be used as a reference for the Barents Sea in general, but specific analyses of the food web structure must take into account when and where prey and predators may actually interact. This information is not provided in the current data set.</p>
        <p><i>C.5.c: data source and qualification</i></p>
        <p>The quality of the information used to document trophic links was graded from 1 to 4, '1', being the most reliable sources of information, and '4' being the least reliable sources of information (Table 1.). </p>
        <p align="left"><span class="smallcaps">Table 1</span>. Coding nomenclature for the trophic links.</p>
        <table border="1" cellspacing="0" cellpadding="0">
          <tbody><tr>
            <td valign="top"><p align="left">Code</p></td>
            <td width="507" valign="top"><p align="left">&nbsp;</p></td>
          </tr>
          <tr>
            <td valign="top"><p align="left">1</p></td>
            <td width="507" valign="top"><p align="left">Link documented by quantitative data and published in peer-reviewed literature</p></td>
          </tr>
          <tr>
            <td valign="top"><p align="left">2</p></td>
            <td width="507" valign="top"><p align="left">Link documented by quantitative data available in databases or published in gray literature</p></td>
          </tr>
          <tr>
            <td valign="top"><p align="left">3</p></td>
            <td width="507" valign="top"><p align="left">Link reported from other sources of information (personal observations, fishermen's report, etc.)</p></td>
          </tr>
          <tr>
            <td valign="top"><p align="left">4</p></td>
            <td width="507" valign="top"><p align="left">Plausible link. It has not been observed but it can be inferred from other observations on related species or other geographic areas</p></td>
          </tr>
        </tbody></table>
        <p align="left">The link was coded '1',when peer reviewed literature was the primary source of information documenting the trophic link between two trophospecies, and when this information was originating from a dedicated observation protocol (e.g., , stomach sampling). The link was coded '2', when the information was not reported in the peer-reviewed literature, but originated from gray literature, e.g., , reports from research institutes or  databases based on a dedicated observation protocol. The link was coded '3', if no dedicated observation protocol was used, but the link was reported in several instances (e.g., , personal observations by scientists or fishermen).</p>
        <p>The link was coded '4', if no primary source of information could be found, but it was thought to be plausible based on observations in other oceanic regions, or of organisms of the same group (e.g., , other species of the same genus).</p>
        <p><b>D. Data limitations and potential enhancements</b></p>
        <p><i>D.1. Definition of nodes</i></p>
        <p><i>D.1.a. geographical and temporal selection criteria</i></p>
        <p>The selection of taxa to be included in the data set relies on selection criteria (section <i>C.4.a.)</i> which include presence of the taxa in the Barents Sea over at least a minimum time and geographic extent (5% of the area). It was often not possible to access robust quantitative data on the spatial and temporal  distribution of taxa so therefore uncertainty remains in the node selection process. Given past changes in the distribution and abundance of many species in the Barents Sea and expected future variations due to climate change, the list of taxa to be included in the topology should be revised and updated in the future. </p>
        <p><i>D.1.b. taxonomy</i></p>
        <p>Taxonomy is an active field of research and the name of all taxa listed in this data set may not remain fixed over time. We have used the WoRMS taxonomy terminology. The inclusion of different taxonomic levels in the same food web is problematic when conducting analysis of the topology. We have provided phylogenetic information so that it is possible to eliminate overlapping taxa when desired. Ultimately, a higher taxonomic resolution could be used to remove taxa above species level.</p>
        <p><i>D.1.c. lower trophic levels</i></p>
        <p>The current topology does not expand to trophic groups below meso-zooplankton, with the exception of diatoms, macroalgae, and mixotrophs. This may be expanded in the future.</p>
        <p><i>D.2. Definition and coding of links</i></p>
        <p><i>D.2.a: coding nomenclature</i></p>
        <p>The coding nomenclature defines the reliability (section C.5.c) of the trophic link. Yet, some of the links need to be revised. To be successful, future attempts to build or complement a topology constructed with the present protocol should ensure sufficient scientific discussion (e.g., , in a workshop) to ensure consensus among the experts involved so that the protocol is well understood and followed.</p>
        <p><i>D.2.b: bias towards predators</i></p>
        <p>Studies on prey–predator interactions are often derived from observation on the predator, typically by conducting stomach analyses. Therefore the reported trophic links are biased towards predatory groups that are well studied, such as commercial fish species, as well as well-studied birds and mammals. For less well studied groups, e.g., , benthic invertebrates, the number of nodes and trophic links may be under-reported.</p>
        <p><i>D.3 Exhaustiveness</i></p>
        <p>Although we have tried to compile a food web topology that is as exhaustive as possible, the current data set is necessarily limited by the joint expertise of the authors. It is expected that the dissemination of the current data set will lead to feedback and improvement through corrections and additions from scientists active in this field.</p>
<p>&nbsp;</p>
        <p class="sectionHeader">Class III. Data set status and accessibility</p>
        <p>
          <b>A. </b><b>Latest update:</b> xxx.
        </p>
        <p>This is the first version of this data set. There is currently no archiving procedure or repository for this data set other than the current publication.</p>
        <p>B. Latest metadata update: </p>
        <p>This is the first version of the metadata.</p>
        <p>C. Copyright or Proprietary Restrictions: </p>
        <p>This data set is freely available for non-commercial scientific use, given the appropriate scholarly citation.</p>
    <p>D. Contact person: </p>
        <p>Kathrine Michalsen. Institute of Marine  Research, PO Box 1870, 5817 Bergen, Norway, kathrine.michalsen@imr.no.</p>
    <p>&nbsp;</p>
<p class="sectionHeader">Class IV. Data structural descriptors</p>
        <p><b>A. Data set files</b></p>
        <p>There are three data files. The first contains the list of trophospecies, the second the list of links and the third the list of literature references. All files are in ASCII format, where each line contains an individual record and each column an individual descriptor. Lines are separated by carriage return (CR) and columns are separated by tabulations (TAB). The first line of each data table contains the column headers.</p>
        <p><i>A.1. List of trophospecies: SpeciesList.txt</i></p>
        <p><i>A.1.a. file format and size</i></p>
        <p>244 records (rows), 7 descriptors (columns), and headers in the first row.</p>
        <p><i>A.1.b. fields</i></p>
        <p>TROPHOSPECIES: Name of the taxa, in Latin.</p>
        <p>ABBREVIATION: Shortened Latin name.</p>
        <p>PHYLUM_SUBPYLUM: Taxa phylum or sybphylum</p>
        <p>CLASS: Taxa class, if applicable</p>
        <p>ORDER: Taxa order, if applicable</p>
        <p>FAMILY: Taxa family, if applicable</p>
        <p>GROUP: Practical grouping identifier (plankton, benthos, fish, birds, mammals)</p>
        <p><i>A.2. List of trophic links: PairwiseList.txt</i></p>
        <p><i>A.2.a. File format and size</i></p>
        <p>1589 records (rows), 4 descriptors (columns) and headers in the first row.</p>
        <p><i>A.2.b. Fields</i></p>
        <p>PREY: Name of the prey taxa</p>
        <p>PREDATOR: Name of the predator taxa</p>
        <p>CODE: qualification code for the trophic link (see section <i>C.5.c</i>)</p>
        <p>REFERENCE: Author and year of publication. When several references are provided, they are separated by a semicolon (;). This field can be empty.</p>
        <p><i>A.3. Literature references: References.txt</i></p>
        <p><i>A.3.a. File format and size</i></p>
        <p>182 records (rows), 2 descriptors  (columns), header in the first row.</p>
        <p><i>A.3.b. Column headers</i></p>
        <p>AUTHOR_YEAR: Author and year of publication</p>
        <p>FULL_REFERENCE: Full bibliographic  reference</p>
        <p><b>B. Relational fields</b></p>
        <p>The data tables in the three files constitute a relational database. The relations between fields in different tables are as follows:</p>
        <p>[SpeciesList.txt] TROPHOSPECIES : [PairwiseList.txt] PREY</p>
        <p>[SpeciesList.txt] TROPHOSPECIES : [PairwiseList.txt] PREDATOR</p>
        <p>[PairwiseList.txt] REFERENCE : [References.txt] AUTHOR_YEAR</p>
<p>&nbsp;</p>
        <p class="sectionHeader">Class V. Supplemental descriptors</p>
        <p><b>A. Comparison between topologies at different quality level</b></p>
    <p>The total number of taxa in this topology is 244, but the number of links to be retained depends on the link quality level. The acceptable level must be defined by users of the topology on a case-by-case basis. When all quality levels are considered (1–4), there are 1589 links out of a possible theoretical total of 2442&nbsp;=&nbsp;59536, giving a connectance of 0.027 (the connectance is the ratio of the number of observed links over the number of possible links). When only peer reviewed publications are considered, the total number of links is 621 and 52 orphan species (i.e., no prey and no predator link) are left. The connectance declines to 0.017. The effects of considering various qualification levels are summarized in Table 2.</p>
    <p><span class="smallcaps">Table 2</span>. Descriptors of the food web topology (rows) for different levels of data quality. 1–4: all quality levels are considered; 1–3: the first 3 quality levels; 1–2: the first 2 quality levels; 1: only quality level 1. </p>
    <table border="1" cellspacing="0" cellpadding="2">
      <tbody><tr>
        <td nowrap valign="top"><p align="left"><b>Quality of links</b></p></td>
        <td nowrap valign="top"><p align="left"><b>1–4</b></p></td>
        <td nowrap valign="top"><p align="left"><b>1–3</b></p></td>
        <td nowrap valign="top"><p align="left"><b>1–2</b></p></td>
        <td nowrap valign="top"><p align="left"><b>1</b></p></td>
      </tr>
      <tr>
        <td nowrap valign="top"><p align="left">Total number of taxa</p></td>
        <td nowrap valign="top"><p align="left">244 </p></td>
        <td nowrap valign="top"><p align="left">244 </p></td>
        <td nowrap valign="top"><p align="left">244 </p></td>
        <td nowrap valign="top"><p align="left">244 </p></td>
      </tr>
      <tr>
        <td nowrap valign="top"><p align="left">Total number of taxa with 1+ link</p></td>
        <td nowrap valign="top"><p align="left">244</p></td>
        <td nowrap valign="top"><p align="left">225</p></td>
        <td nowrap valign="top"><p align="left">220</p></td>
        <td nowrap valign="top"><p align="left">192</p></td>
      </tr>
      <tr>
        <td nowrap valign="top"><p align="left">Total number of links</p></td>
        <td nowrap valign="top"><p align="left">1589</p></td>
        <td nowrap valign="top"><p align="left">1065</p></td>
        <td nowrap valign="top"><p align="left">1033</p></td>
        <td nowrap valign="top"><p align="left">621</p></td>
      </tr>
      <tr>
        <td nowrap valign="top"><p align="left">Cannibalistic links</p></td>
        <td nowrap valign="top"><p align="left">15</p></td>
        <td nowrap valign="top"><p align="left">11</p></td>
        <td nowrap valign="top"><p align="left">10</p></td>
        <td nowrap valign="top"><p align="left">8</p></td>
      </tr>
      <tr>
        <td nowrap valign="top"><p align="left">Connectance 1 (all taxa)</p></td>
        <td nowrap valign="top"><p align="left">0.027</p></td>
        <td nowrap valign="top"><p align="left">0.018</p></td>
        <td nowrap valign="top"><p align="left">0.017</p></td>
        <td nowrap valign="top"><p align="left">0.010</p></td>
      </tr>
      <tr>
        <td nowrap valign="top"><p align="left">Connectance 2 (taxa with 1+ link)</p></td>
        <td nowrap valign="top"><p align="left">0.027</p></td>
        <td nowrap valign="top"><p align="left">0.021</p></td>
        <td nowrap valign="top"><p align="left">0.021</p></td>
        <td nowrap valign="top"><p align="left">0.017</p></td>
      </tr>
    </tbody></table>
    <p>&nbsp;</p>
    <p><b>B. Comparison with existing food web topologies </b></p>
    <p>In the past, several topologies have been published. However, there has not been detailed documentation on how the data  was collected and validated. We report on two topologies that have been used in the past (Table 3). The topology presented by Dommasnes et al. (2001) was used as the basis for the  construction of a combined Ecopath model for the Barents Sea and Norwegian Sea. It is a simplified topology where many nodes correspond to large species groups. This topology contained 30 taxa and 29 trophic links. More recently, Bodini et al. (2009) constructed a more comprehensive topology to study the cascading effects of species removal in a  food web, a phenomenon described as secondary extinctions. The topology data set is not publicly available but it was kindly provided by the first author. Out of the 254 taxa listed by the authors, 151 had at least one connection to a prey or a predator, which resulted in 1001 trophic links. We found the connectance in the Dommasnes et al. (2001) topology to be much higher (0.201) than in that of Bodini et al. (0.044) and in the present study (0.026). </p>
    <p><span class="smallcaps">Table 3</span>. Descriptors of the food web topology (rows) for the current food web (all quality levels), the food web published by Bodini et al. (2009) and the food web published by Dommasnes et al. (2001).</p>
    <table border="1" cellspacing="0" cellpadding="2">
      <tbody><tr>
        <td nowrap valign="top"><p align="left"><b>Topology</b></p></td>
        <td nowrap valign="top"><p align="left"><b>Present study</b></p></td>
        <td nowrap valign="top"><p align="left"><b>Bodini</b></p></td>
        <td nowrap valign="top"><p align="left"><b>Dommasnes</b></p></td>
      </tr>
      <tr>
        <td nowrap valign="top"><p align="left">Total number of taxa</p></td>
        <td nowrap valign="top"><p align="left">244 </p></td>
        <td nowrap valign="bottom"><p align="left">254</p></td>
        <td nowrap valign="bottom"><p align="left">30</p></td>
      </tr>
      <tr>
        <td nowrap valign="top"><p align="left">Total number of taxa with 1+ link</p></td>
        <td nowrap valign="top"><p align="left">244</p></td>
        <td nowrap valign="bottom"><p align="left">151</p></td>
        <td nowrap valign="bottom"><p align="left">29</p></td>
      </tr>
      <tr>
        <td nowrap valign="top"><p align="left">Total number of links</p></td>
        <td nowrap valign="top"><p align="left">1589</p></td>
        <td nowrap valign="bottom"><p align="left">1001</p></td>
        <td nowrap valign="bottom"><p align="left">619</p></td>
      </tr>
      <tr>
        <td nowrap valign="top"><p align="left">Cannibalistic links</p></td>
        <td nowrap valign="top"><p align="left">15</p></td>
        <td nowrap valign="bottom"><p align="left">1</p></td>
        <td nowrap valign="bottom"><p align="left">4</p></td>
      </tr>
      <tr>
        <td nowrap valign="top"><p align="left">Connectance 1 (all taxa)</p></td>
        <td nowrap valign="top"><p align="left">0.027</p></td>
        <td nowrap valign="bottom"><p align="left">0.016</p></td>
        <td nowrap valign="bottom"><p align="left">0.188</p></td>
      </tr>
      <tr>
        <td nowrap valign="top"><p align="left">Connectance 2 (taxa with 1+ link)</p></td>
        <td nowrap valign="top"><p align="left">0.027</p></td>
        <td nowrap valign="bottom"><p align="left">0.044</p></td>
        <td nowrap valign="bottom"><p align="left">0.201</p></td>
      </tr>
    </tbody></table>
<p>&nbsp;</p>
        <p class="sectionHeader">
            Literature cited
        </p>
        <p>Bodini, A., M. Bellingeri, S. Allesina, and C. Bondavalli. 2009. Using food web dominator trees to catch secondary extinctions in action. Philosophical  Transactions of the Royal Society B: Biological Sciences 364:1725–1731.</p>
        <p>Carmack, E., D. Barber, J. Christensen, R. Macdonald, B. Rudels, and E. Sakshaug. 2006. Climate variability and physical forcing of the food webs and the carbon budget on panarctic shelves. Progress in Oceanography 71:145–181.</p>
    <p>Cohen, E. B. 1989. Ecologists Co-operative Web Bank (ECOWeB™). Version 1.0. Machine  Readable Data Base of Food Webs. Rockefeller University, New York, New York, USA.</p>
        <p>Cohen, J. E., R. A. Beaver, S. H. Cousins, D. L. DeAngelis, L. Goldwasser, K. L.  Heong, R. D. Holt, A. J. Kohn, J. H. Lawton, N. Martinez, R. O'Malley, L. M.  Page, B. C. Patten, S. L. Pimm, G. A. Polis, M. Rejmánek, T. W. Schoener, K. Schoenly, W. G. Sprules, J. M. Teal, R. E. Ulanowicz, P. H. Warren, H. M.  Wilbur, and P. Yodzis. 1993. Improving Food Webs. Ecology 74:252–258.</p>
        <p>de Santana, C. N., A. F. Rozenfeld, P. A. Marquet, and C. M. Duarte. 2013. Topological properties of polar food webs. Marine Ecology Progress Series 474:15–26.</p>
    <p>Dommasnes, A., V. Christensen, B. Ellertsen, C. Kvamme, W. Melle, L. Nøttestad, T. Pedersen, S. Tjelmeland, and D. Zeller. 2001. An Ecopath model for the Norwegian Sea and Barents Sea. 2001: Fisheries Impacts on North Atlantic Ecosystems., University of British Columbia.</p>
    <p>Dunne, J. 2009. Food Webs. Pages 3661–3682 <i>in</i> R. A. Meyers, editor. Encyclopedia of complexity and system science. Springer.</p>
        <p>Goldwasser, L., and J. Roughgarden. 1997. Sampling Effects and the Estimation of Food-Web Properties. Ecology 78:41–54.</p>
        <p>Jordán, F. 2003. Comparability: the key to the applicability of food web research. Applied ecology and environmental research 1:1–18.</p>
        <p>Petchey, O., P. J. Morin, and H. Olff. 2010. The topology of ecological interaction  networks: the state of the art. Pages 7–22 <i>in</i> Community Ecology. Oxford University Press, New York, New York, USA.</p>
        <p>Sakshaug, E., G. Johnsen, and K. M. Kovacs. 2009. Ecosystem - Barents Sea. Tapir academic press, Trondheim, Norway.</p>
        <p>Yodzis, P. 1988. The indeterminacy of ecological interactions as perceived through perturbation experiments. Ecology 69:508–515</p>
        <p>&nbsp;</p>
        <p><span class="sectionHeader"><a name="update"></a>UPDATED METADATA (ADDED AUGUST 2015)</span></p>
        <p>Who eats whom in the Barents Sea: a food web topology from  plankton to whales. <i>Ecology</i> 95:1430. <a href="http://dx.doi.org/10.1890/13-1062.1">http://dx.doi.org/10.1890/13-1062.1</a></p>
        <p>Data set version 2</p>
        <p><b>1. Introduction to the revision of the data – August 2015</b></p>
        <p>This second version of the food web topology for the Barents  Sea was compiled and quality checked by Susanne Kortsch (susanne.kortsch@uit.no)  and Benjamin Planque (benjamin.planque@imr.no).</p>
        <p>The information provided in the original publication of the metadata  remains valid with the exception of the points highlighted below.</p>
        <p>The revised food web topology for the Barents Sea includes  revision of the list of trophospecies, revision of the list of pairwise trophic  interactions, revision of the list of references, addition of a new table to  link pairwise interactions to bibliographic references and modification of the  files structure. New automated data qualification procedures were constructed  and run to ensure consistency within and between files.</p>
        <p><b>2. Revision of the species list</b></p>
        <p>Taxa were carefully reviewed to ensure that they matched the  selection criteria. Taxa that were not in the original data set have been included,  such as the abundant polychaete <i>Myriochele heeri</i>, while rare species  e.g., <i>Chimaera monstrosa </i>have  been deleted. Species that are taxonomically misidentified at sea e.g., <i>Dipturus  batis</i>, species that are not found in the Barents Sea e.g., <i>Calanus helgolandicus </i>and species with a more coastal distribution e.g., <i>Mytilus edilus</i> have been deleted. Species have also  been deleted when it was not possible to find diet information, e.g., <i>Stylocheiron  maximum</i>. Some trophospecies have been aggregated from species level to  genus, because of their morphological similarity and the difficulty to  distinguish them when sampled and counted at sea. The revised species list  consists of 233 taxa.</p>
        <p><b>3. Revision of the pairwise list</b></p>
        <p>The pairwise list has been revised by adding new trophic  links and carefully checking the quality coding of individual links. The  revised pairwise list consists of 2218 trophic links.</p>
        <p><b>4. Revision of the reference list</b></p>
        <p>New references have been added to better document links that  were previously reported or to document links that have been added in the  revised data set. The revised reference list consists of 236 bibliographic  references.</p>
        <p><b>5. Modification of the files structure</b></p>
        <p>The original data set consisted of three tables: the <i>species list, </i>the <i>pairwise list</i> and the <i>reference  list</i>. The updated version contains four tables: the <i>species list</i>, the <i>pairwise  list</i>, the <i>pairwise2reference</i> list  and the <i>reference</i> list. We added the <i>pairwise2reference</i> list because the  original format did not allow for matching between the <i>reference</i> field in the pairwise list and the <i>author_year</i> field in the reference list. There can be one, several  or no bibliographic references attached to a single pairwise interaction. To account  for this, the pairwise list was separated into two related files; the <i>pairwise </i>list that contains every single  pairwise interactions and the <i>pairwise2references</i> list that contains a line for every pair of trophic interaction and reference.  The structure of these files, and the relational fields that connect them, are  described in the sections below.</p>
        <p><b>6. Automated data qualification procedures</b></p>
        <p>The following criteria were checked automatically and corrections  were applied where required</p>
        <ul type="disc">
          <li>Species are unique (i.e., listed only once) in the species list</li>
          <li>Pairwise trophic  interactions are unique (i.e. listed only once) in the pairwise list</li>
          <li>All species listed in the  species list are found at least once in the pairwise list, either as prey or as predator</li>
          <li>All species listed in the  pairwise list, either as prey or as predator, are found in the species       list</li>
          <li>All pairwise interactions (PWKEY) in the pairwise2reference table are found in the pairwise list</li>
          <li>All references (AUTHOR_YEAR) listed in the pairwise2reference table are found in the       reference table</li>
          <li>All references (AUTHOR_YEAR) listed in the reference table are found in the       pairwise2reference table</li>
          <li>In the reference table, names of the first author in the AUTHOR_YEAR field and in the FULL_REFERENCE field match.</li>
        </ul>
    <p><b>7. Data set files</b></p>
        <p>There are four data files. The first contains the list of  trophospecies, the second the list of trophic links, the third the list of associations between individual trophic links and references and the fourth the list of literature references. All files are in ASCII format, where each line contains an individual record and each column an individual descriptor. Lines  are separated by carriage return (CR) and columns are separated by tabulations (TAB). The first line of each data table contains the column headers.</p>
        <blockquote>
          <p>7.1. List of trophospecies: SpeciesList.txt</p>
        </blockquote>
        <p>7.1.a. file format and size</p>
        <p>233 records (rows), 7 descriptors (columns), and headers in the first row.</p>
        <p>7.1.b. fields</p>
        <p>TROPHOSPECIES: Name of the taxa, in Latin. (Primary key for this table)</p>
        <p>ABBREVIATION: Shortened Latin name.</p>
        <p>PHYLUM_SUBPYLUM: Taxa phylum or sybphylum</p>
        <p>CLASS: Taxa class, if applicable</p>
        <p>ORDER: Taxa order, if applicable</p>
    <p>FAMILY: Taxa family, if applicable</p>
        <p>GROUP: Practical grouping identifier (plankton, benthos, fish, birds, mammals)</p>
    <blockquote>
          <p>7.2. List of trophic links: PairwiseList.txt</p>
    </blockquote>
        <p>7.2.a. File format and size</p>
        <p>2218 records (rows), 4 descriptors (columns) and headers in  the first row.</p>
        <p>7.2.b. Fields</p>
        <p>PWKEY: Pairwise identification Key- (Primary key for this table). This is constructed by joining the abbreviations of the predator and  prey species</p>
        <p>PREY: Name of the prey taxa</p>
    <p>PREDATOR: Name of the predator taxa</p>
        <p>CODE: qualification code for the trophic link (see section <i>C.5.c in the original metadata description</i>)</p>
    <blockquote>
          <p>7.3. trophic interaction &amp; references: Pairwise2References.txt</p>
    </blockquote>
        <p>7.3.a. File format and size</p>
        <p>2126 records (rows), 2 descriptors (columns), header in the first row.</p>
        <p>7.3.b. Column headers</p>
    <p>PWKEY: Pairwise identification Key (note that this is not a primary key for this table)</p>
        <p>AUTHOR_YEAR: Author(s) and year of publication</p>
        <blockquote>
          <p>7.4. Literature references: References.txt</p>
        </blockquote>
        <p>7.4.a. File format and size</p>
        <p>236 records (rows), 2 descriptors (columns), header in the first row.</p>
        <p>7.4.b. Column headers</p>
    <p>AUTHOR_YEAR: Author(s) and year of publication</p>
        <p>FULL_REFERENCE: Full bibliographic reference</p>
        <blockquote>
          <p>7.5. Relational fields</p>
        </blockquote>
        <p>The data tables in the three files constitute a relational database. The relations between fields in different tables are as follows:</p>
        <p>[SpeciesList.txt] TROPHOSPECIES : [PairwiseList.txt] PREY</p>
        <p>[SpeciesList.txt] TROPHOSPECIES : [PairwiseList.txt]  PREDATOR</p>
    <p>[PairwiseList.txt] PWKEY : [Paiwise2References.txt] PWKEY</p>
        <p>[Paiwise2References.txt] AUTHOR_YEAR : [References.txt]  AUTHOR_YEAR</p>
        <p><b>8. Comparison with the original food web topology</b></p>
        <p>The total number of taxa in this topology is 233. The number of links to be  retained for a particular analysis depends on the link quality level. The acceptable level must be defined by users of the topology on a case-by-case  basis. When all quality levels are considered (1–4), there are 2218 links out of a possible theoretical total of 233^2&nbsp;=&nbsp;54289, giving a  connectance of 0.041 (the connectance is the ratio of the number of observed links over the number of possible links). When only peer-reviewed publications are considered, the total number of links is 1258 and 10 orphan species (i.e., species without any trophic link link) are left. The connectance declines to 0.026. The effects of considering various qualification levels are summarized in Table 1.</p>
    <p><span class="smallcaps">Table 1</span>. Descriptors of the food web topology (rows) for different levels of data quality. 1–4: all quality levels are considered; 1–3: the first 3 quality levels; 1–2: the first 2 quality levels; 1: only quality level 1. The upper part of the table repeats information provided in the original metadata file. The lower part of the table indicates update for the  revised food web topology.</p>
        <table border="1" cellspacing="0" cellpadding="0">
          <tbody><tr>
            <td nowrap valign="top"><p align="left"><b>Original data set (version 1)</b></p></td>
            <td nowrap valign="top"><p align="left">&nbsp;</p></td>
            <td nowrap valign="top"><p align="left">&nbsp;</p></td>
            <td nowrap valign="top"><p align="left">&nbsp;</p></td>
            <td nowrap valign="top"><p align="left">&nbsp;</p></td>
          </tr>
          <tr>
            <td nowrap valign="top"><p align="left">Quality of links (Data V1)</p></td>
            <td nowrap valign="top"><p align="left">1–4</p></td>
            <td nowrap valign="top"><p align="left">1–3</p></td>
            <td nowrap valign="top"><p align="left">1–2</p></td>
            <td nowrap valign="top"><p align="left">1</p></td>
          </tr>
          <tr>
            <td nowrap valign="top"><p align="left">Total number of taxa</p></td>
            <td nowrap valign="top"><p align="left">244 </p></td>
            <td nowrap valign="top"><p align="left">244 </p></td>
            <td nowrap valign="top"><p align="left">244 </p></td>
            <td nowrap valign="top"><p align="left">244 </p></td>
          </tr>
          <tr>
            <td nowrap valign="top"><p align="left">Total number of taxa with 1+ link</p></td>
            <td nowrap valign="top"><p align="left">244</p></td>
            <td nowrap valign="top"><p align="left">225</p></td>
            <td nowrap valign="top"><p align="left">220</p></td>
            <td nowrap valign="top"><p align="left">192</p></td>
          </tr>
          <tr>
            <td nowrap valign="top"><p align="left">Total number of links</p></td>
            <td nowrap valign="top"><p align="left">1589</p></td>
            <td nowrap valign="top"><p align="left">1065</p></td>
            <td nowrap valign="top"><p align="left">1033</p></td>
            <td nowrap valign="top"><p align="left">621</p></td>
          </tr>
          <tr>
            <td nowrap valign="top"><p align="left">Cannibalistic links</p></td>
            <td nowrap valign="top"><p align="left">15</p></td>
            <td nowrap valign="top"><p align="left">11</p></td>
            <td nowrap valign="top"><p align="left">10</p></td>
            <td nowrap valign="top"><p align="left">8</p></td>
          </tr>
          <tr>
            <td nowrap valign="top"><p align="left">Connectance 1 (all taxa)</p></td>
            <td nowrap valign="top"><p align="left">0.027</p></td>
            <td nowrap valign="top"><p align="left">0.018</p></td>
            <td nowrap valign="top"><p align="left">0.017</p></td>
            <td nowrap valign="top"><p align="left">0.010</p></td>
          </tr>
          <tr>
            <td nowrap valign="top"><p align="left">Connectance 2 (taxa with 1+ link)</p></td>
            <td nowrap valign="top"><p align="left">0.027</p></td>
            <td nowrap valign="top"><p align="left">0.021</p></td>
            <td nowrap valign="top"><p align="left">0.021</p></td>
            <td nowrap valign="top"><p align="left">0.017</p></td>
          </tr>
          <tr>
            <td nowrap valign="top"><p align="left"><b>Revised data set    (version 2)</b></p></td>
            <td nowrap valign="top"><p align="left">&nbsp;</p></td>
            <td nowrap valign="top"><p align="left">&nbsp;</p></td>
            <td nowrap valign="top"><p align="left">&nbsp;</p></td>
            <td nowrap valign="top"><p align="left">&nbsp;</p></td>
          </tr>
          <tr>
            <td nowrap valign="top"><p align="left">Quality of links</p></td>
            <td nowrap valign="top"><p align="left">1–4</p></td>
            <td nowrap valign="top"><p align="left">1–3</p></td>
            <td nowrap valign="top"><p align="left">1–2</p></td>
            <td nowrap valign="top"><p align="left">1</p></td>
          </tr>
          <tr>
            <td nowrap valign="top"><p align="left">Total number of taxa</p></td>
            <td nowrap valign="top"><p align="left">233 </p></td>
            <td nowrap valign="top"><p align="left">233</p></td>
            <td nowrap valign="top"><p align="left">233</p></td>
            <td nowrap valign="top"><p align="left">233</p></td>
          </tr>
          <tr>
            <td nowrap valign="top"><p align="left">Total number of taxa with 1+ link</p></td>
            <td nowrap valign="top"><p align="left">233</p></td>
            <td nowrap valign="top"><p align="left">226</p></td>
            <td nowrap valign="top"><p align="left">224</p></td>
            <td nowrap valign="top"><p align="left">223</p></td>
          </tr>
          <tr>
            <td nowrap valign="top"><p align="left">Total number of links</p></td>
            <td nowrap valign="top"><p align="left">2218</p></td>
            <td nowrap valign="top"><p align="left">1672</p></td>
            <td nowrap valign="top"><p align="left">1601</p></td>
            <td nowrap valign="top"><p align="left">1258</p></td>
          </tr>
          <tr>
            <td nowrap valign="top"><p align="left">Cannibalistic links</p></td>
            <td nowrap valign="top"><p align="left">26</p></td>
            <td nowrap valign="top"><p align="left">22</p></td>
            <td nowrap valign="top"><p align="left">21</p></td>
            <td nowrap valign="top"><p align="left">16</p></td>
          </tr>
          <tr>
            <td nowrap valign="top"><p align="left">Connectance 1 (all taxa)</p></td>
            <td nowrap valign="top"><p align="left">0.041</p></td>
            <td nowrap valign="top"><p align="left">0.031</p></td>
            <td nowrap valign="top"><p align="left">0.029</p></td>
            <td nowrap valign="top"><p align="left">0.023</p></td>
          </tr>
          <tr>
            <td nowrap valign="top"><p align="left">Connectance 2 (taxa with 1+ link)</p></td>
            <td nowrap valign="top"><p align="left">0.041</p></td>
            <td nowrap valign="top"><p align="left">0.032</p></td>
            <td nowrap valign="top"><p align="left">0.032</p></td>
            <td nowrap valign="top"><p align="left">0.026</p></td>
          </tr>
        </tbody></table>
        

        <!-- Content stops here!  Don't delete anything after this! -->
        <hr>
        <p class="center">[<a href="default.php">Back to E095-124</a>]</p>
    

<!--?php mysql_close($con); ?-->
</body></html>